challenges/classes.js

```diff
@@ -1,7 +1,41 @@
 // 1. Copy and paste your prototype in here and refactor into class syntax.
+// function CuboidMaker(props) {
```

Comment:
I know you just copy-pasted to have the reference, but upon completing the refactor, delete this commented-out code. It just adds clutter and since you did the class correctly, you don't need this anymore.